dbaas/account/models.py

```diff
@@ -41,13 +53,73 @@ def get_query_set(self):
         return User.objects.filter(id__in=[user.id for user in Team.users_without_team()])
 
 
+class Organization(BaseModel):
+    name = models.CharField(
+        verbose_name = 'Name',
+        help_text='Organization name.',
+        max_length=100, null=False, blank=False)
+    grafana_orgid = models.CharField(
+        verbose_name = 'Grafana Org ID',
+        help_text='External organization id. Used on grafana dashboard.',
+        max_length=10, null=True, blank=True)
+    grafana_hostgroup = models.CharField(
+        verbose_name= 'Grafana Hostgroup',
+        help_text='External grafana Hostgroup. Used to retrinct access on metrics.',
+        max_length=50, null=True, blank=True)
+    grafana_datasource = models.CharField(
+        verbose_name = 'Grafana Datasource',
+        help_text='Datasource used on external organization.',
+        max_length=50, null=True, blank=True)
+    grafana_endpoint = models.CharField(
+        verbose_name = 'Grafana Endpoint',
+        help_text='Endpoint used on external organization.',
+        max_length=255, null=True, blank=True)
+    external = models.BooleanField(
+        verbose_name = "External",
+        default=False,
+        help_text='Whether the organization is external. If checked, all fields become mandatory.')
+
+    def __unicode__(self):
+        return self.name
+
+    @property
+    def databases(self):
```

Comment:
See if this code make sense: return Database.objects.filter(team__organization=self.id)
Or if has related. self.database_set.all()